tests/10apidoc/33room-members.pl

```diff
@@ -408,15 +408,38 @@ sub matrix_create_and_join_room
 
 sub room_fixture
 {
+   my ( $user_fixture, %args ) = @_;
+
+   fixture(
+      requires => [ $user_fixture ],
+
+      setup => sub {
+         my ( $user ) = @_;
+
+         matrix_create_room( $user, %args )->then( sub {
+            my ( $room_id ) = @_;
+
+            Future->done( $room_id );
+         });
+      }
+   );
+}
+
+push @EXPORT, qw( magic_room_fixture );
+
+sub magic_room_fixture
+{
    my %args = @_;
 
+   my $magic = delete $args{magic};
```

Comment:
I've looked this up and down and I don't see any use of `$magic` here, nor any calls passing it in. Could this line be deleted therefore?